File: src/main/java/com/paragon/client/managers/social/Relationship.java
package com.paragon.client.managers.social;

import net.minecraft.util.text.TextFormatting;

/**
 * @author Wolfsurge
 */
public enum Relationship {
    /**
     * Do not attack them
     */
    FRIEND(TextFormatting.GREEN),
    /**
     * Attack them, but do display them as neutral instead of enemy
     */
    NEUTRAL(TextFormatting.GRAY),
    /**
     * Attack them and display them as an enemy
     */
    ENEMY(TextFormatting.RED);

    private TextFormatting textFormatting;

    Relationship(TextFormatting formatting) {
        this.textFormatting = formatting;
    }

    public TextFormatting getTextFormatting() {
        return textFormatting;
    }
}

File: src/main/kotlin/com/paragon/client/managers/social/Relationship.kt
package com.paragon.client.managers.social

import net.minecraft.util.text.TextFormatting

/**
 * @author Surge
 */
enum class Relationship(val textFormatting: TextFormatting) {

    /**
     * Do not attack them
     */
    FRIEND(TextFormatting.GREEN),

    /**
     * Attack them, but do display them as neutral instead of enemy
     */
    NEUTRAL(TextFormatting.GRAY),

    /**
     * Attack them and display them as an enemy
     */
    ENEMY(TextFormatting.RED);

}